### ISSUE TITLE
area5 eval miou<20%

### ISSUE THREAD

**Issue by gxushine** [mellody11/RandLA-Net-Pytorch-New#5]
感谢您的工作。请问，我在win10上训练100次，训练精度一直在提高，area5评估精度一直小于20%,原因是什么？
测试环境win10/cuda11.2/python3.9/bach_size=1
![image](https://user-images.githubusercontent.com/38464816/259611651-0f346dcf-1ad1-49b2-9983-06830732f45d.jpeg)

**Comment by mellody11** [comment]
您好，感谢对本复现工作的关注。对于您遇到的问题，希望您能提供更多的信息。
例如：
1. 您是否修改过代码？
2. 在其他Area上是否有这种情况？

**Comment by YoungJackson** [comment]
> 感谢您的工作。请问，我在win10上训练100次，训练精度一直在提高，area5评估精度一直小于20%,原因是什么？ 测试环境win10/cuda11.2/python3.9/bach_size=1 ![image](https://user-images.githubusercontent.com/38464816/259611651-0f346dcf-1ad1-49b2-9983-06830732f45d.jpeg)

不要在win上训练，可能出现无法预知的错误

**Comment by mellody11** [comment]
您好，所以您现在问题解决了？

So you have resolved the problem now?

**Comment by YoungJackson** [comment]
解决了

**Comment by mellody11** [comment]
ok，感谢您提供的信息。

Ok, thanks for your information.

**Comment by gxushine** [comment]
RandLA-Net模型在bach_size=1的时候会产生过拟合现象。所以要确保bach_size>=2，此时可以降低num_points来解决GPU内存不足的问题。 另外原作者和本项目的RandLA-Net模型都可以在windows和ubuntu下运行，需要注意的是在windows下运行时”最近邻搜索knn_search“的结果错误的，需要 参考   原作者代码后面的回复 或者 参考Open3D实现RandLA-Net模型的代码 来解决。再次感谢博主的工作！

**Comment by mellody11** [comment]
很棒，谢谢您提供的信息和方法！

Great, thank you for the information and methods you provided!

**Comment by scream68** [comment]
Regarding the issue with KNN in the Windows environment, how can it be solved? I would like to ask for your advice. Alternatively, could you share the links you referred to at that time? @gxushine

**Comment by gxushine** [comment]
> Regarding the issue with KNN in the Windows environment, how can it be solved? I would like to ask for your advice. Alternatively, could you share the links you referred to at that time? [@gxushine](https://github.com/gxushine)

[Open3d-ml](https://github.com/isl-org/Open3D-ML) integrates Randla-Net and has two versions: TensorFlow and PyTorch. You can refer to how they implemented KNN using Open3D. @scream68

**Comment by scream68** [comment]
I have already implemented it locally and applied it to the segmentation of industrial dense point cloud products. However, it seems that it cannot meet the accuracy requirements. Anyway, thank you for your answer. @gxushine
